actionview/lib/action_view/digestor.rb

```diff
@@ -70,18 +70,11 @@ def tree(name, finder, partial = false, seen = {})
       end
 
       private
-        def find_template(finder, *args)
-          name = args.first
-          prefixes = args[1] || []
-          partial = args[2] || false
-          keys = args[3] || []
-          options = args[4] || {}
+        def find_template(finder, *args, **options)
```

Comment:
Thanks. This was my first suggestion: https://github.com/rails/rails/pull/32612#discussion_r184844300

but i thought, having both splat args and kwargs should do the trick. I'll read more about warning, and change ruby bugtracker if there are any related issues.

Reply:
My point is it is only passed 5 args, not variable and not passed `options`.

Reply:
But it actually forwards everything bypassing all the args. However explicit args seems to be the only way to avoid warning in 2.5.0, even if it'll be fixed in ruby later.